react/index.js

```diff
@@ -1,112 +1,292 @@
+  return {
+    title: 'editor.search-result.title',
+    description: 'editor.search-result.description',
+    type: 'object',
+    properties: {
+      maxItemsPerPage: {
+        title: 'editor.search-result.maxItemsPerPage.title',
+        type: 'number',
+        default: DEFAULT_MAX_ITEMS_PER_PAGE,
+      },
+      hiddenFacets: {
+        title: 'editor.search-result.hiddenFacets',
+        type: 'object',
+        isLayout: true,
+        properties: {
+          brands: {
+            title: 'editor.search-result.hiddenFacets.brands',
+            type: 'boolean',
+            isLayout: true,
+          },
+          categories: {
+            title: 'editor.search-result.hiddenFacets.categories',
+            type: 'boolean',
+            isLayout: true,
+          },
+          priceRange: {
+            title: 'editor.search-result.hiddenFacets.priceRange',
+            type: 'boolean',
+            isLayout: true,
+          },
+          specificationFilters: {
+            title: 'editor.search-result.hiddenFacets.specificationFilters',
+            type: 'object',
+            isLayout: true,
+            properties: {
+              hideAll: {
+                title: 'editor.search-result.hiddenFacets.specificationFilters.hideAll',
+                type: 'boolean',
+                isLayout: true,
+              },
+              hiddenFilters: {
+                type: 'array',
+                isLayout: true,
+                items: {
+                  title: 'editor.search-result.hiddenFacets.specificationFilters.hiddenFilter',
+                  type: 'object',
                   isLayout: true,
-                  items: {
-                    title: 'editor.search-result.hiddenFacets.specificationFilters.hiddenFilter',
-                    type: 'object',
-                    isLayout: true,
-                    properties: {
-                      name: {
-                        title: 'editor.search-result.hiddenFacets.specificationFilters.hiddenFilter.name',
-                        type: 'string',
-                        isLayout: true,
-                      },
+                  properties: {
+                    name: {
+                      title: 'editor.search-result.hiddenFacets.specificationFilters.hiddenFilter.name',
+                      type: 'string',
+                      isLayout: true,
                     },
                   },
                 },
               },
             },
           },
         },
-        summary: {
-          title: 'editor.search-result.summary.title',
-          type: 'object',
-          properties: ProductSummary.getSchema(props).properties,
-        },
       },
-    }
-  }
-
-  render() {
-    return <SearchResultInfiniteScroll {...this.props} />
+      summary: {
+        title: 'editor.search-result.summary.title',
+        type: 'object',
+        properties: ProductSummary.getSchema(props).properties,
+      },
+      showMore: {
+        type: 'boolean',
```

Comment:
Change to be an enum like a `type`, that can have several option (infinite scroll, pagination or show more button)

Reply:
👍